### ISSUE TITLE
ServerAuth

### ISSUE THREAD

**Issue by SuperKali** [poggit-orphanage/office#13]
## Request to add a plugin on Poggit Release
I am writing to request that an old plugin be added onto Poggit Release. Details are as follows:

#### Plugin Name

ServerAuth

#### Link to GitHub repo

https://github.com/EvolSoft/ServerAuth/tree/master/ServerAuth

#### Is it in the pre-Alpha_1.4 format?

not work on pocketmine alpha7

**Comment by SOF3** [comment]
@TheDiamondYT1 Are you related to ServerAuth?

**Comment by SOF3** [comment]
@Flavius12 Please reply if you are not inactive.

**Comment by TheDiamondYT1** [comment]
@SOF3 I am in the EvolSoft organization, so yes. I am guessing the OP wants ALPHA7/8 support (is ALPHA8 even out yet?). I never touched ServerAuth much because it looked big and scary 😂

**Comment by TheDiamondYT1** [comment]
Please test the latest commit

**Comment by lukeeey** [comment]
i just came across this again and i should probably give my new account admin access and update ServerAuth at some point.

**Comment by SOF3** [comment]
Do people still use auth plugins these days? 2-factor authentication should be a better alternative.

**Comment by lukeeey** [comment]
True, but with a good auth plugin then servers without xbox authentication can appear.

**Comment by Flavius12** [comment]
Can players still play on servers without Xbox live access?
Newest MCPE versions do not allow to play on servers without Xbox live access, am I wrong?

**Comment by dktapps** [comment]
You can use a basic tunnel proxy to join servers as if they are LAN servers - like this one: https://gist.github.com/dktapps/4fa8a6df15e851f6b58d01df36bd0aa3 (you don't need XBLA to join a LAN server, so you can trick the client into thinking the server is on LAN)

Additionally XBLA is not foolproof in PocketMine at the moment because protocol encryption is not implemented.

**Comment by lukeeey** [comment]
@dktapps You can also use client mods. I made one and joined a few servers and they all required xbox auth except one 🙄
